### PR TITLE
BGDIINF_SB-1362: Improved test code and added items endpoint test

### PR DESCRIPTION
**Pull request by ltshb**
Instead of redoing the db initialization in each test, moved the code in a package, database.py and reused it in tests.
    
Added items endpoints test and also the collections list endpoint test.

### CODE REVIEW

File: app/tests/database.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    '''Create a dummy collection db object for testing
+    '''
+    collection = Collection.objects.create(
+        crs=["http://www.opengis.net/def/crs/OGC/1.3/CRS84"],
+        created='2020-10-28T13:05:10.473602Z',
+        updated='2020-10-28T13:05:10.473602Z',
+        description='This is a description',
+        start_date=None,
+        end_date=None,
+        extent=[200000, 100000, 200001, 100005],
```
Comment (hansmannj) [inline app/tests/database.py:21]: This is not compatible with the most recent collection model.
No separate start_ and end_date anymore, extent is now a JSONField covering temporal and spatial extent.

File: app/tests/database.py [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+        collection_name='ch.swisstopo.pixelkarte-farbe-pk200.noscale',
+        item_type='Feature',
+        license='test',
+        stac_extension=get_default_stac_extensions(),
+        stac_version="0.9.0",
+        summaries_eo_gsd=[10.1, 20.3, 30.44],
+        summaries_proj=[1, 4, 22],
+        geoadmin_variant=['blubb', 'blabb', 'blibb'],
```
Comment (hansmannj) [inline app/tests/database.py:29]: Now aggregated to one single summaries-JSONField

**Review by hansmannj — CHANGES_REQUESTED** [review]
Nice approach, just a few minor comments :+1:

File: app/tests/database.py (hunk 2 above)
Comment (ltshb) [reply app/tests/database.py:29]: Well it depends on who is merging first :-). Each PR must be compatible with the `develop` branch and not on other PR. So Either I merged it now (this changes needs to be part of your PR then) or I wait that you merge the `Collection` serializer and do the changes within this PR :-)

**Review by hansmannj — APPROVED** [review]

**Comment by ltshb** [comment]
This work has been push within #57, wasn't intended sorry :-(

**Pull request closed without merging**